osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -160,11 +161,11 @@ private void playSample(T value)
 
             var channel = sample.Play();
 
-            channel.Frequency.Value = 1 + NormalizedValue * 0.2f;
+            channel.Frequency.Value = 1 + NormalizedValue * 0.2f + RNG.NextDouble(0.02f);
```

Comment:
This is going to be a bit of a nitpick but I'd probably make this

```suggestion
            channel.Frequency.Value = 1 + NormalizedValue * 0.2f + RNG.NextDouble(0.02f) - 0.01f;
```

The reason for that is that `RNG.NextDouble(x)` returns random values from the interval `[0, x]`. So as-is, the distribution of possible frequencies is the continuous range of `[initial_value, initial_value + 0.02f]`. Shifting that back down by `0.01f` yields a distribution of `[initial_value - 0.01f, initial_value + 0.01f]` and so the distribution is centered around the initial value.

Reply:
The change you're proposing will actually change the audible sound. @nekodex can you confirm you only intended the random portion to be in an upwards direction here?

Reply:
The goal here was just to add a tiny bit of randomisation to the otherwise linear pitch scaling to make things sound a bit more natural, the range of that randomisation was less important.

I'll try the suggested change, but honestly I'm not sure it would result in an appreciable difference to the listener?

Reply:
It won't be audible for sure, the current code was just sticking out to my inner sense of symmetry, which is why I pre-empted the comment by saying it's a nitpick.

`HoverSounds` already does things in a way similar to what I proposed, for whatever that's worth:

https://github.com/ppy/osu/blob/c2759121b9fb6acafd37d5a72720a1bef0fc1ccf/osu.Game/Graphics/UserInterface/HoverSounds.cs#L39

Reply:
@bdach did you mean to adjust the initial value to keep the same range?

Reply:
No, my intention was that range would change. Compare the `HoverSounds` snippet above, which uses a range of [0.98, 1.02] for frequency, which is centered at 1. What this pull does right now would be more akin to [1, 1.04]. But honestly not really even fussed about this that much anymore.